frontend/pages/home_page.js

```diff
@@ -27,20 +35,47 @@ export default class HomePage extends Component {
     this.fetchProducts();
   }
 
+  productStatusIncludesFilter = (string) => {
+    const { complete, incomplete, pendingReview } = this.state;
+    string = string.toLowerCase();
```

Comment:
I haven't looked at how this method is being used yet, i've only seen the method declaration so far and I don't understand it, probably for 2 reasons:

1. i dont know what 'string' is referring to or what it means
2. i dont know what 'ret' is referring to or what it means

i do understand that string is a string, and i understand that ret is a boolean whose value is determined by whether the string is equal to the status.

Reply:
what we discussed:
This method shouldn't worry about checking if there's a filter, it should just check that the filter matches the status:
```
const productMarkup = [];
data.forEach((product, index) => {
  if (filter && !statusMatchesFilter(product)) return;
  productMarkup.push(<Component key={index} productData={product} />)
});
return productMarkup;
```